src/main.js

```diff
@@ -7,29 +7,26 @@ var form = document.querySelector(".form-view");
 var mainCoverSection = document.querySelector(".main-cover");
 var savedViewSection = document.querySelector(".saved-view");
 var savedSectionGrid = document.querySelector(".saved-covers-section");
-//buttons
+var savedCover = document.querySelector(".mini-cover");
+// buttons QS
 var ownCoverButton = document.querySelector(".make-new-button");
 var randomButton = document.querySelector(".random-cover-button");
 var saveCoverButton = document.querySelector(".save-cover-button");
 var homeButton = document.querySelector(".home-button");
 var viewSavedButton = document.querySelector(".view-saved-button");
 var makeBookButton = document.querySelector(".create-new-book-button");
-var savedCover = document.querySelector(".mini-cover");
-var overlay = document.querySelector(".overlay");
-//form inputs
+// form inputs QS
 var coverField = document.querySelector(".user-cover");
 var titleField = document.querySelector(".user-title");
 var desc1Field = document.querySelector(".user-desc1");
 var desc2Field = document.querySelector(".user-desc2");
 
-// We've provided a few variables below
 var savedCovers = [
-  // new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
+   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover = {};
 
-
-// Add your event listeners here 👇
+//event listeners
 window.addEventListener("load", randomBook);
```

Comment:
This works! Another way you could do this is

window.onload = randomBook();

Same deal, just thought you might like to see an alternative way as well.